kube/services/jobs/gentestdata-job.yaml

```diff
@@ -83,6 +83,8 @@ spec:
             GEN3_TEST_PROJECT|-value: "test"-|
           - name: MAX_EXAMPLES
             GEN3_MAX_EXAMPLES|-value: "10"-|
+          - name: SUBMISSION_ORDER
+            GEN3_MAX_EXAMPLES|-value: "submitted_unaligned_reads"-|
```

Comment:
you probably want `GEN3_SUBMISSION_ORDER` here to allow command-line override?

Reply:
😮 Oh, how does that work?
You have some cool mechanism that seamlessly acts upon `GEN3_` prefixed parameters? (that's cool)

Let me push a commit...

Reply:
also - update the documentation comment at the top ...